bagh/bagh.json

```diff
@@ -5,6 +5,14 @@
   "status": "beschikbaar",
   "version": "0.0.1",
   "crs": "EPSG:28992",
+  "versioning": {
+    "request_parameter": "versie",
+    "version_field_name": "volgnummer",
+    "pk_field_name": "identificatie",
+    "temporal": {
```

Comment:
Dit zijn dan eigenlijk de temporele dimensies... dus `dimensions`?

Reply:
Dus dan samen wordt het:
```
"temporal": {
  "identifier:" "volgnummer",
  "dimensions: {
    "geldigOp": ["begin geldigheid", "eind geldigheid"]
  }
}
```
 
Ik dacht dat `versie` parameter was conform DSO specs.

Reply:
Dat is hem dan inderdaad. Kort en bondig.